ircbot/commands_impl.go

```diff
@@ -69,7 +69,50 @@ func LastPlayed(e Event) error {
 	return nil
 }
 
-func StreamerQueue(e Event) error       { return nil }
+func StreamerQueue(e Event) error       {
+	// Get queue from streamer
+	songQueue, err := e.Bot.Streamer.Queue(e.Context())
+	if err != nil {
+		return err
+	}
+
+	// If the queue is empty then we're done
+	if len(songQueue) == 0 {
+		e.EchoPublic("No queue at the moment")
```

Comment:
you can use `NewPublicError(nil, "No queue at the moment")` here to get an error that the handler knows to send to the channel